spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -281,17 +281,17 @@ class SpdxExpressionTest : WordSpec() {
             "split-up compound expressions with AND or OR operator but not ones with WITH operator" {
                 "GPL-2.0-or-later WITH Classpath-exception-2.0 AND MIT"
                     .decompose() shouldContainExactlyInAnyOrder listOf(
-                        "GPL-2.0-or-later WITH Classpath-exception-2.0",
-                        "MIT"
-                    )
+                    "GPL-2.0-or-later WITH Classpath-exception-2.0",
```

Comment:
Indentation got broken here.

Reply:
Done. Added a commit to align formatting with the auto formatter.